src/VerifierAlgorithm.ts

```diff
@@ -50,7 +54,7 @@ export function verifyES256K(
     signer = verifyRecoverableES256K(data, signature, ethAddressKeys)
   }
 
-  if (!signer) throw new Error('Signature invalid for JWT')
+  if (!signer) throw new Error(formatErrorMessage(data, signature))
```

Comment:
Error messages should not indicate the data being verified.
These errors can bubble up and leak information.

Reply:
Reason I did this was because as I've been using the library the error messages are quite unhelpful. Thought this would help with that :)

Reply:
right, but that can also be done at an application level.
In your situation, wherever you call the method you know your JWT so it would be easier to catch the error and display it in a safe manner to aid your debugging.

The same cannot be said about all applications using this lib and since it's a common mistake to bubble up errors it can easily lead to information leaks.